README.md

```diff
@@ -1,4 +1,4 @@
-Systers Portal [![Build Status](https://travis-ci.org/systers/portal.svg?branch=master)](https://travis-ci.org/systers/portal) [![Coverage Status](https://coveralls.io/repos/systers/portal/badge.png?branch=master)](https://coveralls.io/r/systers/portal?branch=master)
+Systers Portal [![Build Status](https://travis-ci.org/systers/portal.svg?branch=master)](https://travis-ci.org/systers/portal) [![Coverage Status](https://coveralls.io/repos/github/systers/portal/badge.svg?branch=master)](https://coveralls.io/github/systers/portal?branch=master)
```

Comment:
Clicking on coverage should redirect to https://coveralls.io/r/systers/portal?branch=master. Please change the link https://coveralls.io/github/systers/portal?branch=master to https://coveralls.io/r/systers/portal?branch=master

Reply:
Done :+1: